modules/local/star_align.nf

```diff
@@ -61,6 +76,8 @@ process STAR_ALIGN {
     cat <<-END_VERSIONS > versions.yml
     "${task.process}":
         star: \$(STAR --version | sed -e "s/STAR_//g")
+        samtools: \$(echo \$(samtools --version 2>&1) | sed 's/^.*samtools //; s/Using.*\$//')
+        gawk: \$(echo \$(gawk --version 2>&1) | sed 's/^.*GNU Awk //; s/, .*\$//')
```

Comment:
Why awk here?

Reply:
No, this was just me being nice to users not having to pull multiple containers to run STAR since these modules are local to the pipeline anyway. `gawk` and `samtools` are actually required by the `STAR_GENOMEGENERATE` module so I thought I would re-use the container.